redis/redis.conf

```diff
@@ -0,0 +1 @@
+databases 100
```

Comment:
Could you add a newline at the end of file?

Reply:
@ajardin fixed